### DISCUSSION TITLE
do you know more youtube like apps??

### DISCUSSION THREAD

**Discussion by idontveapc** [nyas1/Material-You-app-list#66] [Q&A]
i mean as honista and such... not a new build but rebuild of official YouTube app.

**Comment by nyas1** [comment]
Yes, as @TeaEndsAcronyms said, you can use YouTube Revanced with extended patches by inotia00. You can patch it yourself with [Revanced Manager](https://github.com/revanced/revanced-manager) with help of [Tutorial ](https://github.com/TeaEndsAcronyms/Material-You-Apps/releases/tag/Tutorials!-4.0) they made. Or you can use prepatched apks.
![photo_6336930491416033505_y](https://github.com/nyas1/Material-You-app-list/assets/85426900/378184d9-4347-4d3b-8750-7052c4c3282f)
![photo_6336930491416033506_y](https://github.com/nyas1/Material-You-app-list/assets/85426900/07b58dd6-ba42-454d-8a49-8ec8fd849108)

**Reply by ghost** [reply]
Thanks @nyas1 :)

**Comment by idontveapc** [comment]
i have been using revanced and other frontends or clients (like newpipe) they lack customisability execpt light and dark theme options.
